src/models/Act.ts

```diff
@@ -0,0 +1,6 @@
+export const Act: Object = {
+  name: String,
```

Comment:
types almost always should be using `string`, `boolean`, the lower case primitives are preferred. Again, I'll let you look into this :)
https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string

Reply:
Apologies. I'd had actually already updated this but did not push it with the correct changes.